docs/modules/ROOT/pages/references/dns-naming-scheme.adoc

```diff
@@ -0,0 +1,99 @@
+= DNS naming scheme
+
+NOTE: The DNS naming scheme fulfills the usability requirements documented in xref:appuio-cloud:ROOT:references/quality-requirements/usability/dns-naming-scheme.adoc[DNS Naming scheme].
+
+== Base domain
+
+The base domain is `appuio.cloud`.
+
+Common services that are zone independent are served on the third level.
+Services bound to a specific {zone}, will be served through the fourth level where the third level is the zone's identifier.
+Apps served through am {zone}'s default domain use the subdomain `apps` of a zone's base domain.
+
+.Known hosts and schemes of 'appuio.cloud'
+|===
+| Hostname | Description
+
+|`appuio.cloud`, `www.appuio.cloud`
+| Hosting the marketing homepage for {product}
+
+|`id.appuio.cloud`
+|identity provider for {product}
+
+|`console.appuio.cloud`
```

Comment:
" console" might be confusing because it interferes with OpenShift console. I would prefer something a different name to make it clear, this are different "consoles". Like control, portal, my.

Reply:
```suggestion
|`my.appuio.cloud`
```

I agree with the reasoning. Let's use this suggestion for now. As we don't know yet the details about this application we can later change the URL again when we know more... We're still weeks / months away from having something to deploy anyways.